defaultValues.js

```diff
@@ -0,0 +1,5 @@
+DEFAULT_PORT   = 8000;
```

Comment:
Since I can't other `DEFAULT_PORT` in this file, it must be a global variable, or you create it right there since there is no `const` or `let`. Avoid using global variables and enable [no-implicit-globals](https://eslint.org/docs/rules/no-implicit-globals) ESlint rule.